contents/blog/remote-culture.md

```diff
@@ -0,0 +1,238 @@
+showTitle: true
+hideAnchor: true
+---
+
+![Remote Work Banner](../images/blog/remote-culture/remote-banner.jpg) 
+
+
+<small class="centered-text">Photo by Corinne Kutz</small>
+
+
+<br />
+
+2020 has been a tough year. 
+
+Both in our personal and professional lives, we were all forced to make adjustments. Some small, some big.
+
+For many companies that can afford to do so, 2020 has become the year of "work from home". 
+
+A quote from an acquaintance of mine, an executive at a large European firm, sums it up quite well:
+
+<blockquote class="blog-quote">
+
+For a long time we discussed a day of work from home for a select group of employees. Now, we had to put the majority of the workforce on remote work overnight.
+
+</blockquote><br />
+
+Remote work is nothing new. In fact, it has been increasing steadily in popularity over the last decade, as the world continues to become increasingly connected by the internet.
+
+<br />
+
+![Google Trends Remote Work](../images/blog/remote-culture/remote-trends.png)
+<small class="centered-text">Interest in "remote work" in the US from 2004-Present. Source: [Google Trends](https://medium.com/r/?url=https%3A%2F%2Ftrends.google.com%2Ftrends%2Fexplore%3Fdate%3Dall%26geo%3DUS%26q%3Dremote%2520work)</small>
+
+<br />
+
+
+However, many have now been "forced" to learn about it, and more, have to actually implement it. 
+
+So, given that our team combined has a few decades of remote work experience, and we're in the process of figuring out our own remote culture here at PostHog, we thought we'd share some of what we learned along the way.
+
+This is by no means an objective list, and far from a tutorial. Rather, it is a compilation of lessons we've learned as an all-remote company, as we continue to try to strengthen that culture ourselves.
+
+So here are some points we find important, in no particular order of importance.
+
+## Socializing
+
+<br />
+
+![Virtual Coffee Bot](../images/blog/remote-culture/virtual-coffee.png) 
+
+<br />
+
+
+When you work with a team in an office, you're able to chat with people throughout the day, make jokes during a coffee break, and go out for beers after a long day. 
+
+And even if your team does not socialize much after-hours, the slightest social interactions with individuals around us are often biologically good for us. They make us feel better.
+
+When you're all-remote, well, that's not a thing. As such, you have to make up for it somehow. 
+
+At PostHog, we're actually still trying to figure this part out, but we've introduced some concepts that have been extremely helpful:
+
+#### Life Story Fridays
+Every Friday, someone from the team shares their life story with everyone else. This is a great way to know who you're working with beyond the professional level, as well as learn some cool things about your teammates along the way. For example, we just recently found out one of our developers ranks in the top 5 for Pi digit memorization in the UK. 
+
+#### Virtual Coffee
```

Comment:
coffee, #random are somewhat generic ideas. Perhaps change life story fridays to "understand the actual people you work with" as a header, then mainly talk about how weird life story fridays are, then chuck in these ocuple of other bits in one sentence?